charts/kyverno-policies/values.yaml

```diff
@@ -0,0 +1,2 @@
+podSecurityStandard: baseline
```

Comment:
Need some context on why this and the template boilerplate exists.

Reply:
The upstream repo has different categories -> https://github.com/kyverno/policies/tree/main/pod-security  Let's decide if we need the same

Reply:
okay